assets/src/blocks/CarouselHeader/Caption.js

```diff
@@ -0,0 +1,64 @@
+const { RichText } = wp.blockEditor;
+const { __ } = wp.i18n;
+
+export const Caption = ({ slide, index, setAttributes, isSelected, slides }) => {
+  const changeSlideAttribute = (slideAttributeName, index, value, slides) => {
+    let slidesCopy = JSON.parse(JSON.stringify(slides));
+    slidesCopy[index][slideAttributeName] = value;
+    setAttributes({slides: slidesCopy});
+  }
+
+  return <div className="carousel-caption">
+    <div className="caption-overlay"></div>
+    <div className="container main-header">
+      <div className="row">
+        <div className="col">
+          <div className="carousel-captions-wrapper">
+            <RichText
```

Comment:
Can you please remove the formatting options from the RichText components, by adding `allowedFormats={[]}`? In preparation for https://jira.greenpeace.org/browse/PLANET-5780 🙂